src/app/GitUI/Editor/Diff/DiffHighlightService.cs

```diff
@@ -317,18 +370,18 @@ private void AddExtraPatchHighlighting(IDocument document)
                 diffContentOffset = 4;
             }
 
-            MarkDifference(document, linesRemoved, linesAdded, diffContentOffset);
+            MarkInlineDifferences(document, linesRemoved, linesAdded, diffContentOffset);
```

Comment:
I prefer to not mark the filenames in the headers.
Removed in my branch feature/difflineinfo-coloring. (The filename in the header is also not parsed now, will need to be added.)

Reply:
Removed